fathom/Cargo.toml

```diff
@@ -32,7 +32,7 @@ string-interner = "0.14.0"
 termsize = "0.1.6"
 
 [build-dependencies]
-lalrpop = "0.19.5"
+lalrpop = { git = "https://github.com/kmeakin/lalrpop", branch = "raw-identifiers" }
```

Comment:
Will obviously need to be published if we want to eventually publish Fathom, but that’s a long way off, so I think it’s ok for now!